### PR TITLE
Always run in UTC + iCal name

### PR DESCRIPTION
**Pull request by simong**
Couple of minor fixes

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -18,6 +18,9 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+// Always run in UTC, no matter what
+process.env.TZ = 'UTC';
```
Comment (nicolaasmatthijs) [inline app.js]: Is this required? Wouldn't we always be working with UTC timestamp when using date objects and "getTime()"?

File: node_modules/gh-tests/lib/util.js [hunk 2]
```diff
@@ -596,10 +596,14 @@ var parseIcalCalendar = module.exports.parseIcalCalendar = function(calendar, ca
...
-        // assert.ok(calendar.fields);
-        // assert.ok(calendar.fields.PRODID);
+        // Assert we return a name for this calendar
+        var prodId = _.find(calendar.rawFields, {'key': 'PRODID'});
+        assert.ok(prodId);
+        assert.ok(prodId.value);
+
+        var calName = _.find(calendar.rawFields, {'key': 'X-WR-CALNAME'});
+        assert.ok(calName);
+        assert.ok(calName.value);
```
Comment (nicolaasmatthijs) [inline node_modules/gh-tests/lib/util.js]: Do we have enough information to assert that the name is the expected one?

**Comment by nicolaasmatthijs** [comment]
Finished review. Back to @simong for follow-up

File: app.js (hunk 1 above)
Comment (simong) [inline app.js]: This is only an issue when your machine isn't running in UTC (fwiw, all our servers are).
New date objects default to the `TZ` timezone when created. It becomes a real issue when serialising a calendar to ical (the cozy-ical library doesn't seem to handle this terribly well).

(fwiw, we had to do it in OAE as well: https://github.com/oaeproject/Hilary/blob/master/node_modules/oae-util/lib/oae.js#L62)

**Comment by simong** [comment]
followed up, back to @nicolaasmatthijs

**Comment by nicolaasmatthijs** [comment]
Left one final question.

It also looks like there's a test failure, although it's probably not related to this PR:

```
1) Oauth authentication DAO verify two access tokens with same token value cannot be created:
      Uncaught AssertionError: (key: "updatedAt") false == true
      + expected - actual
      -false
      +true
      
  
  2) Oauth authentication Access Tokens verify exchange authorization code:
      Uncaught AssertionError: (key: "updatedAt") false == true
      + expected - actual
      -false
      +true
```

**Comment by nicolaasmatthijs** [comment]
Back to @simong

**Comment by simong** [comment]
Those OAuth tests have been resolved, assigning to @mrvisser for review

**Comment by mrvisser** [comment]
@simong there are still a bunch of scripts that don't set UTC at the top (in `etc/scripts...`). Shouldn't they?

**Comment by mrvisser** [comment]
Back to @simong for feedback

**Comment by simong** [comment]
I'll address that in the rollover branch, back to @mrvisser

**Pull request merged**